odo/backends/sql.py

```diff
@@ -172,7 +172,9 @@ def discover_typeengine(typ):
     if isinstance(typ, (sa.NUMERIC, sa.DECIMAL)):
         return datashape.Decimal(precision=typ.precision, scale=typ.scale)
     if isinstance(typ, (sa.String, sa.Unicode)):
-        return datashape.String(typ.length, typ.collation)
+        return datashape.String(typ.length, None)
```

Comment:
do youy know if there is a way to get the encoding off the type? I remember looking at this in the past but I don't remember why I didn't fix this. I agree that passing the collation for the encoding is strictly incorrect though.